x-pack/platform/plugins/shared/streams/server/lib/streams/attachments/attachment_client.ts

```diff
@@ -426,33 +530,13 @@ export class AttachmentClient {
       },
     });
 
-    const attachmentDocuments: AttachmentDocument[] = attachmentsResponse.hits.hits.map(
-      ({ _source }) => _source
-    );
+    // Convert attachment documents to attachment links
+    const attachmentLinks: AttachmentLink[] = attachmentsResponse.hits.hits.map(({ _source }) => ({
```

Comment:
Do we have to use source or can we use fields instead? 

Help me quickly understand the attachment link vs the UUID. Can you give 2 examples of each?